relayer/src/worker/packet.rs

```diff
@@ -267,17 +267,32 @@ fn acknowledgment_metrics(path: &Packet, summary: &RelaySummary) {
 #[cfg(feature = "telemetry")]
 fn timeout_metrics(path: &Packet, summary: &RelaySummary) {
     use ibc::events::IbcEvent::TimeoutPacket;
-    let count = summary
+    let mut count = 0;
+    summary
         .events
         .iter()
-        .filter(|e| matches!(e, TimeoutPacket(_)))
-        .count();
+        .filter(|event| matches!(event, TimeoutPacket(_)))
+        .for_each(|e| {
+            let height = e.height().revision_height(); // TODO : remove height argument
```

Comment:
Let's remove the height arguments everywhere in this PR?

Reply:
I thought maybe opening a new PR would be good as well. If removing height introduces new bugs, breaks some tests or if it becomes useful to revert it, having two separate PRs could be helpful. What do you think about it ?